src/api/Search/types.ts

```diff
@@ -40,6 +40,7 @@ export interface SearchResponse {
   totalHits?: number; // one among totalHits and moreResults is defined
   moreResults?: boolean;
```

Comment:
you also need to add the itemTypeCountLimit to the configuration interfaces in the rest-client